### DISCUSSION TITLE
can renovate version AMIs?

### DISCUSSION THREAD

**Discussion by caphrim007** [renovatebot/renovate#10273] [Request Help] [answered]
Hi,

I haven't specifically looked into this, I'm just asking the community before I do.

Has anyone used renovate to keep AMIs updated? In our infrastructure we usually just say "latest" and terraform (for instance) will do a query to determine what the latest AMI is.

Just wondering if this might be possible to determine the "latest" using similar filters via renovate or if it's not currently possible.

Thanks in advance!

**Comment by rarkins** [comment, marked as answer]
Not currently supported, but we'd like to. Here's a related feature request: https://github.com/renovatebot/renovate/issues/8178

**Comment by BrunnerLivio** [comment]
A Pull Request just got merged and there is experimental support.

https://github.com/renovatebot/renovate/pull/11869
